docs/sources/user_guide/data/iris_data.ipynb

```diff
@@ -71,13 +71,14 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Dimensions: 150 x 4\n",
-      "\n",
-      "Header: ['sepal length', 'sepal width', 'petal length', 'petal width']\n",
-      "1st row [ 5.1  3.5  1.4  0.2]\n"
+     "ename": "ModuleNotFoundError",
```

Comment:
Looks like there's an issue with your mlxtend installation. I recommend running

```
python setup.py develop`
```

in the main folder of your mlxtend branch again, then try to run `python make_api.py` again, followed by running the notebook again.